content/master/concepts/providers.md

```diff
@@ -89,28 +106,280 @@ crossplane-stable/crossplane \
+{{<hover label="offline" line="7">}}packagePullPolicy: Never{{</hover>}}.
+
+```yaml {label="offline"}
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: offline-provider-aws
+spec:
+  package: provider-aws
+  packagePullPolicy: Never
+```
+
+### Installation options
+
+Providers support multiple configuration options to change installation related
+settings. 
+
+#### Provider pull policy
+
+Use a {{<hover label="pullpolicy" line="6">}}packagePullPolicy{{</hover>}} to
+define when Crossplane should download the Provider package to the local
+Crossplane package cache.
+
+The `packagePullPolicy` options are: 
+* `IfNotPresent` - (**default**) Only download the package if it isn't in the cache.
+* `Always` - Check for new packages every minute and download any matching
+  package that isn't in the cache.
+* `Never` - Never download the package. Packages are only installed from the
+  local package cache. 
+
+{{<hint "tip" >}}
+The Crossplane 
+{{<hover label="pullpolicy" line="6">}}packagePullPolicy{{</hover>}} works
+like the Kubernetes container image 
+[image pull policy](https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy).  
+
+Crossplane supports the use of tags and package digest hashes like
+Kubernetes images. 
+{{< /hint >}}
+
+For example, to `Always` download a given Provider package use the 
+{{<hover label="pullpolicy" line="6">}}packagePullPolicy: Always{{</hover>}}
+configuration. 
+
+```yaml {label="pullpolicy",copy-lines="6"}
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: provider-aws
+spec:
+  packagePullPolicy: Always
+# Removed for brevity
+```
+
+#### Revision activation policy
+
+The `Active` package revision
+is the package controller actively reconciling resources. 
+
+By default Crossplane sets the most recently installed package revision as 
+`Active`.
+
+Control the Provider upgrade behavior with a
+{{<hover label="revision" line="6">}}revisionActivationPolicy{{</hover>}}.
+
+The {{<hover label="revision" line="6">}}revisionActivationPolicy{{</hover>}} 
+options are:
+* `Automatic` - (**default**) Automatically use the latest Provider version
+  available in the cache. 
+* `Manual` - Require the current Provider in use to be manually set. 
+
+For example, to change the upgrade behavior to require manual upgrades, set 
+{{<hover label="revision" line="6">}}revisionActivationPolicy: Manual{{</hover>}}.
+
+```yaml {label="revision"}
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: provider-aws
+spec:
+  revisionActivationPolicy: Manual
+# Removed for brevity
+```
+
+{{<hint "important" >}}
+Crossplane only upgrades a Provider if a newer version is in the package cache.   
+By default the Crossplane [`packagePullPolicy`](#provider-pull-policy) doesn't
+download new Provider versions, even if they're available.
+{{< /hint >}}
+
+Read the [Provider upgrade](#upgrade-a-provider) section for
+more information on the use of package revisions.
```

Comment:
This section is still inaccurate as detailed https://github.com/crossplane/docs/pull/566#discussion_r1367879599

> Crossplane only upgrades a Provider if a newer version is in the package cache.   
By default the Crossplane [`packagePullPolicy`](#provider-pull-policy) doesn't
download new Provider versions, even if they're available.

For example, the `revisionActivationPolicy` has nothing to do with the upgrades.